### PR TITLE
[1.x] Uses PHP Native Type Declarations 🐘

### PR DESCRIPTION
**Pull request by nunomaduro**
Please check https://github.com/laravel/laravel/pull/6010 for details about this pull request.

### CODE REVIEW

File: stubs/UpdateUserPassword.php [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 namespace App\Actions\Fortify;
 
+use App\Models\User;
```
Comment (zakriyarahman) [inline stubs/UpdateUserPassword.php:5]:
```suggestion
use Illuminate\Foundation\Auth\User as Authenticatable;
```

File: stubs/ResetUserPassword.php [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 namespace App\Actions\Fortify;
 
+use App\Models\User;
```
Comment (zakriyarahman) [inline stubs/ResetUserPassword.php:5]:
```suggestion
use Illuminate\Foundation\Auth\User as Authenticatable;
```

File: stubs/UpdateUserProfileInformation.php [hunk 3]
```diff
@@ -2,6 +2,7 @@
 
 namespace App\Actions\Fortify;
 
+use App\Models\User;
```
Comment (zakriyarahman) [inline stubs/UpdateUserProfileInformation.php:5]:
```suggestion
use Illuminate\Foundation\Auth\User as Authenticatable;
```

File: stubs/UpdateUserProfileInformation.php [hunk 4]
```diff
@@ -12,11 +13,9 @@ class UpdateUserProfileInformation implements UpdatesUserProfileInformation
     /**
      * Validate and update the given user's profile information.
      *
-     * @param  mixed  $user
-     * @param  array  $input
-     * @return void
+     * @param  array<string, string>  $input
      */
-    public function update($user, array $input)
+    public function update(User $user, array $input): void
```
Comment (zakriyarahman) [inline stubs/UpdateUserProfileInformation.php:18]:
```suggestion
    public function update(Authenticatable $user, array $input): void
```

File: stubs/UpdateUserProfileInformation.php [hunk 5]
```diff
@@ -44,11 +43,9 @@ public function update($user, array $input)
     /**
      * Update the given verified user's profile information.
      *
-     * @param  mixed  $user
-     * @param  array  $input
-     * @return void
+     * @param  array<string, string>  $input
      */
-    protected function updateVerifiedUser($user, array $input)
+    protected function updateVerifiedUser(User $user, array $input): void
```
Comment (zakriyarahman) [inline stubs/UpdateUserProfileInformation.php:48]:
```suggestion
    protected function updateVerifiedUser(Authenticatable $user, array $input): void
```

File: tests/Fixtures/Models/User.php [hunk 6]
```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Models;
```
Comment (driesvints) [inline tests/Fixtures/Models/User.php:3]: @nunomaduro I'd use `Laravel\Fortify\Tests\Models` for this.

**Pull request merged**